applications/poisson/gaussian/application.h

```diff
@@ -321,8 +321,8 @@ class Application : public ApplicationBase<dim, n_components, Number>
     pp_data.error_data.time_control_data.is_active = true;
     pp_data.error_data.analytical_solution.reset(new Solution<dim>());
 
-    std::shared_ptr<PostProcessorBase<dim, Number>> pp;
-    pp.reset(new PostProcessor<dim, Number>(pp_data, this->mpi_comm));
+    std::shared_ptr<PostProcessorBase<dim, n_components, Number>> pp;
+    pp.reset(new PostProcessor<dim, n_components, Number>(pp_data, this->mpi_comm));
```

Comment:
We should remote the `new` instances sometime and replace them by `std::make_shared` calls.